addons/hr_recruitment/views/res_config_settings_views.xml

```diff
@@ -52,6 +52,18 @@
                                     <widget name="iap_buy_more_credits" service_name="sms" hide_service="1"/>
                                 </div>
                             </div>
+                            <div class="col-xs-12 col-md-6 o_setting_box" title="use OCR to fill data from a picture of the CV or the file itself">
+                                <div class="o_setting_left_pane">
+                                    <field name="module_hr_recruitment_extract" widget="upgrade_boolean"/>
+                                </div>
+                                <div class="o_setting_right_pane" id="recruitment_extract_settings">
+                                    <label string="CV Digitalization (OCR)" for="module_hr_recruitment_extract"/>
+                                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific."/>
+                                    <div class="text-muted">
+                                        Digitalize your CV with OCR and Artificial Intelligence
```

Comment:
We realized recently that the correct word to use is "digitize" instead of "digitalize".
You can check task [2862606](https://www.odoo.com/web#id=2862606&cids=1&model=project.task&view_type=form) for the explanation.